BuildAssets/create-linux-packages.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Remove existing packages, fpm doesnt like to overwrite
+rm *.{deb,rpm}
```

Comment:
This logs a nasty error to stderr if the files dont exist. Be nice if we could suppress that.

Reply:
The only way I know is
```suggestion
rm -f *.{deb,rpm}
```